include/odp/api/spec/packet.h

```diff
@@ -2290,6 +2291,61 @@ int odp_packet_tx_compl_request(odp_packet_t pkt, const odp_packet_tx_compl_opt_
  */
 int odp_packet_has_tx_compl_request(odp_packet_t pkt);
 
+/**
+ * Packet proto stats options.
+ */
+typedef struct odp_packet_proto_stats_opt_t {
+	/** Packet proto stats object handle
+	 *
+	 * Stats in the packet proto stats object will be updated.
+	 */
+	odp_proto_stats_t stat;
+
+	/** Octet counter 0 adjust */
+	int oct_count0_adj;
```

Comment:
Other packet API offsets and lengths are uint32_t type. Similarly, this could be int32_t. In theory, "int" could be 16 bits, but in practice it is 32 bits. So, using int32_t would results the same size, but would document it explicitly.